lightning-rapid-gossip-sync/src/processing.rs

```diff
@@ -75,6 +72,8 @@ impl<NG: Deref<Target=NetworkGraph<L>>, L: Deref> RapidGossipSync<NG, L> where L
 
 			let node_id_1_index: BigSize = Readable::read(read_cursor)?;
 			let node_id_2_index: BigSize = Readable::read(read_cursor)?;
+
+			// Is this only because we truncate max node ids read? Why not do a >0 bounds check as well :shrug:
```

Comment:
it preëmpts an array index out of bounds error. >=0 is unnecessary because bigsize only supports unsigned ints under the hood

Reply:
got it, makes sense on the bigsize -- thanks